x-pack/plugins/uptime/public/components/common/charts/ping_histogram.test.tsx

```diff
@@ -48,18 +58,13 @@ describe('PingHistogram component', () => {
     },
   };
 
-  it('shallow renders the component without errors', () => {
```

Comment:
I decided this test wasn't worth keeping. It asserted that the `PingHistogramComponent` contained the props that were passed to it, and not much else.

Rendering these components doesn't yield SVG or any other method of ensuring the data is rendered by Elastic Charts, but we also should not be testing those capabilities in our repo.